### DISCUSSION TITLE
DocumentWrite Labs: Technical Writing and Content Marketing Resource for Open Source Projects

### DISCUSSION THREAD

**Discussion by epicfaace** [PlaintextGroup/oss-virtual-incubator#34] [Proposal Discussion]
Discuss https://github.com/PlaintextGroup/oss-virtual-incubator/blob/main/proposals/content_resource_center.md -- proposal by @pkafei

**Comment by epicfaace** [comment]
Some initial thoughts:
- I think this could fit well within Plaintext's goal of "broaden the concept of OSS / digital infrastructure to include aspects that may not have been traditionally funded (such as code refactoring, maintenance, community management, and documentation writing)"
- Main question I have is -- how can we best set up DocumentWrite Labs for success so that projects begin to actively use and contribute to it? We wouldn't want this content just to be created in a vacuum, but would rather that we already have folks lined up who are excited to use it. Some potential thoughts / options I'm thinking of:
  - hire a community manager responsible for outreach to the community
  - Launch a "cohort" program in which a couple of OSS projects have regular training / guidance from DocumentWrite Labs to improve their specific project's documentation
  - do some research, before launching DocumentWrite Labs, and identify a couple of OSS projects who would really like to have the kind of guides that DocumentWrite Labs would be writing
  - would it make sense to focus on a specific vertical of projects to start?

**Reply by brainwane** [reply]
Some previous resources that might be helpful to refer to:

[*Docs for Developers: An Engineer's Field Guide to Technical Writing*](https://docsfordevelopers.com/)

the [Season of Docs program](https://developers.google.com/season-of-docs/)

[a successful initiative in the Zulip project to provide English writing tutoring to new contributors](https://www.harihareswara.net/posts/2019/gsocoutreachy-mentoring-orgs-consider-giving-applicants-english-tutoring/)
